var/spack/repos/builtin/packages/parallel-netcdf/package.py

```diff
@@ -12,12 +12,15 @@ class ParallelNetcdf(AutotoolsPackage):
     Unidata's NetCDF."""
 
     homepage = "https://trac.mcs.anl.gov/projects/parallel-netcdf"
-    url      = "http://cucis.ece.northwestern.edu/projects/PnetCDF/Release/parallel-netcdf-1.6.1.tar.gz"
+    url      = "http://cucis.ece.northwestern.edu/projects/PnetCDF/Release/parallel-netcdf-1.10.0.tar.gz"
```

Comment:
This change should not be needed; can you remove it?

Reply:
OK.